.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Want to contribute? Great! Make sure you've checked all these boxes before creating your PR:
+
+- [ ] Keep your PR rebased to master.
```

Comment:
Actually I think this should now be removed. With Wally rebasing should only happen when there are merge conflicts.